db/tests/value_tests.rs

```diff
@@ -14,11 +14,24 @@ extern crate mentat_db;
 extern crate ordered_float;
 extern crate rusqlite;
 
-use mentat_core::{TypedValue, ValueType};
-use mentat_db::db::TypedSQLValue;
+use std::rc::Rc;
+
 use ordered_float::OrderedFloat;
+
 use edn::symbols;
 
+use mentat_core::{TypedValue, ValueType};
+use mentat_db::db::TypedSQLValue;
+
+fn typed_keyword(ns: &'static str, name: &'static str) -> TypedValue {
```

Comment:
Shouldn't we be exposing and using these helpers on `TypedValue` more generally?